apps/trading-e2e/src/integration/trading-deal-ticket-order.cy.ts

```diff
@@ -100,9 +100,8 @@ describe('deal ticker order validation', { tags: '@smoke' }, () => {
         .within(() => {
           cy.get('[data-state="closed"]').should(
             'have.text',
-            'Total margin available'
+            'Total margin available100,000.01 tDAI'
```

Comment:
I realise I'm supposed to review the graphql, but I can't help than think it's weird theire's no spaces between available and 100,....